apps/backend/db_patches/0168_XpressEmailSetting.sql

```diff
@@ -0,0 +1,12 @@
+DO
+$$
+BEGIN
+  IF register_patch('0168_XpressEmailSetting.sql', 'Chi Kai Lam', 'Xpress email setting for recipients SAMPLE_SAFETY_EMAIL and TECHNIQUE_SCIENTISTS_EMAIL', '2025-01-13') THEN
+    BEGIN
+      INSERT INTO settings (settings_id, settings_value, description) VALUES ('SAMPLE_SAFETY_EMAIL', '', 'Email address for the sample safety.'); 
+      INSERT INTO settings (settings_id, settings_value, description) VALUES ('TECHNIQUE_SCIENTISTS_EMAIL', '', 'Email address for the Xpress technique scientist.');
```

Comment:
I'm not sure how it will work to have technique scientists as a static setting. The email needs to be sent only to the scientists belonging to the technique the proposal is submitted to, and we don't know which technique that is until the proposal has been submitted. If the setting has every technique scientist in it, it would alert scientists in other techniques that don't care about the proposal (not to mention require us to maintain another list).